### PR TITLE
Add test frontend for better designing

### PR DESCRIPTION
**Pull request by TooAngel**
The frontend has three views

- / - The frontpage, no magic involved, easy to work on

- /dashboard - The overview of the repositories and some details of the
Pull Requests
To be able to work on this a github oauth key was necessary and
reloading took a long time, due to all the github requests

- /:org/:repo/pull/:pull_number - Detailed overview of the pull request
information
Viewable without key, long loading times because of the calculations

To make it easier to work on the design/UX/UI of `/dashboard` and
`/:org/:repo/pull/:pull_number` a new path `/test` is introduced which
mocks the requests to github.
`/test/dashboard` to show the dashboard
`/test/:org/:repo/pull/:pull_number` to show the pull request view

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ Copy `.env-example` to `.env` and add your environment variables.
 docker-compose up
 ```
 
+## Frontend
+
+The frontent has three views:
```
Comment (IanFindlay) [inline README.md:37]:
```suggestion
The frontend has three views:
```

File: README.md [hunk 2]
```diff
@@ -32,6 +32,19 @@ Copy `.env-example` to `.env` and add your environment variables.
...
+- `/` the frontpage
+- `/dashboard` a dashboard with an overview of the repositories and a button to
+enable world driven for the repository
```
Comment (IanFindlay) [inline README.md:40]:
```suggestion
enable World Driven for the repository
```

File: README.md [hunk 3]
```diff
@@ -32,6 +32,19 @@ Copy `.env-example` to `.env` and add your environment variables.
...
+- `/:org/:repo/pull/:pull_number` A detailed calculation breakdown for the pull
+request
+
+To be able to easily work on the different views test endpoints are available:
+- `/test/dashboard` - for the dashboard
+- `/test/:org/:repo/pull/:pull_number` - for the pull request view
+
 ## Production environment
 
 The worlddriven auto merge service is hosted on heroku, tested via CircleCI,
```
Comment (IanFindlay) [inline README.md:50]:
```suggestion
The worlddriven auto-merge service is hosted on heroku, tested via CircleCI,
```

File: README.md (hunk 1 above)
Comment (TooAngel) [reply README.md]: thx, fixed

File: README.md (hunk 2 above)
Comment (TooAngel) [reply README.md]: thx, fixed

File: README.md (hunk 3 above)
Comment (TooAngel) [reply README.md]: thx, fixed

**Review by IanFindlay — APPROVED** [review]

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**